app/client/jsx/JitsiVideo.jsx

```diff
@@ -84,9 +84,13 @@ class JitsiVideo extends Component {
                 toolbarButtons = _.without(this.toolbarButtons, 'microphone')
             }
 
-            const domain = 'party.gbre.org/jitsi/'
-            // const domain = 'meet.jit.si';
+            let domain = `${window.location.host}/jitsi/`
+            if (Config.jitsiServerUrl) {
+                domain = Config.jitsiServerUrl
+            }
+
             const options = {
+                noSSL: true,
```

Comment:
ideally we pull this out to the config, but this is def an improvement, so can leave for followup work

Reply:
that's easy enough--it should be `true` for local and `false` on prod?

Reply:
correct!